src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5594,6 +5600,10 @@ public static final double getSmithsnessModifier(
     }
 
     for (AdventureResult effect : effects) {
+      int effectId = effect.getEffectId();
+      if (effectId != EffectPool.VIDEO_GAMES
```

Comment:
comment here please or we're going to forget to update it.

or could we pre-cache the effect ids that have smithsness by looking up modifiers at the start of the maximization session (or mafia session, when first requested)?

Reply:
Sure, I'll add a comment. I wanted to hardcode the code to evaluate whether or not it's a smithsness effect so it's as fast as possible. I think it's unlikely effects will be added in the future, but not impossible of course.